pyanaconda/ui/gui/spokes/keyboard.py

```diff
@@ -384,6 +385,9 @@ def _wait_ready(self):
         self._ready = True
         hubQ.send_ready(self.__class__.__name__, False)
 
+        # report that the keyboard spoke initialization has been finished
```

Comment:
The `completed` sounds more appropriate here. With the finished it felt to me like the spoke has completed it's work completely not just initialized itself.

Thank you for change.